ProjectSettings/RiderScriptEditorPersistedState.asset

```diff
@@ -0,0 +1,15 @@
+%YAML 1.1
```

Comment:
I don't think this is needed is it?

Reply:
Like ignore it? I can try that, I just as a rule thought new project settings files should be added